packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -749,4 +749,22 @@ class RenderIndexedStack extends RenderStack {
     super.debugFillProperties(properties);
     properties.add(IntProperty('index', index));
   }
+
+  @override
+  List<DiagnosticsNode> debugDescribeChildren() {
+    final List<DiagnosticsNode> children = <DiagnosticsNode>[];
+    if (firstChild != null) {
+      int i = 0;
+      RenderObject? child = firstChild;
+      do {
```

Comment:
nit:
you can just use 
while (child != null) 
and avoid having to check that firstChild != null

Reply:
@jacob314 Hey, sorry for the delay.
You're right. I have removed the check. :)